cmd/upgrade.go

```diff
@@ -157,8 +157,13 @@ func (uc *upgradeCmd) loadCluster() error {
 
 	if uc.containerService.Properties.IsAzureStackCloud() {
 		writeCustomCloudProfile(uc.containerService)
-		err = uc.containerService.Properties.SetAzureStackCloudSpec()
-		if err != nil {
+		isUpgrade := true
```

Comment:
Looks like these vars are not needed any longer, or they should be used for the input to the construction of the struct below.